src/components/AvatarCropModal/libs/imageManipulator/index.native.js

```diff
@@ -0,0 +1,24 @@
+import RNImageManipulator from '@oguzhnatly/react-native-image-manipulator';
```

Comment:
This should go in package.json

Reply:
What do you mean by that? It is already in package.json
<img width="489" alt="image" src="https://user-images.githubusercontent.com/44551710/168135851-7bee9213-5e69-4ba1-8e79-c939f3affc3e.png">